app/uk/gov/hmrc/agentclientrelationships/services/FindRelationshipsService.scala

```diff
@@ -31,27 +31,24 @@ import scala.concurrent.{ExecutionContext, Future}
 class FindRelationshipsService @Inject()(des: DesConnector, val metrics: Metrics) extends Monitoring {
 
   def getItsaRelationshipForClient(
-    clientId: MtdItId)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Option[ItsaRelationship]] =
-    des.getActiveClientItsaRelationships(clientId)
+    clientId: MtdItId)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Option[ActiveRelationship]] =
+    des.getActiveClientRelationships(clientId)
 
   def getItsaRelationshipForClient(
-    nino: Nino)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Option[ItsaRelationship]] =
+    nino: Nino)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Option[ActiveRelationship]] =
     for {
       mtdItId       <- des.getMtdIdFor(nino)
-      relationships <- des.getActiveClientItsaRelationships(mtdItId)
+      relationships <- des.getActiveClientRelationships(mtdItId)
     } yield relationships
 
   def getVatRelationshipForClient(
```

Comment:
all the above 3 methods can be simplied with just one ?

Reply:
another ticket but can rearrange for now